core/hosting/HostWithCoreClrHost/src/coreclrhost.h

```diff
@@ -0,0 +1,57 @@
+// Retrieved from https://github.com/dotnet/coreclr/blob/master/src/coreclr/hosts/inc/coreclrhost.h
```

Comment:
This really is unfortunate.  This should be coming from a coreclr hosting sdk, which does not yet exist.

Reply:
Agreed